diff --git a/library/ssl_cli.c b/library/ssl_cli.c
index 0c62c34..b1d5c3a 100644
--- a/library/ssl_cli.c
+++ b/library/ssl_cli.c
@@ -551,7 +551,8 @@ static int ssl_write_client_hello( ssl_context *ssl )
                    buf[4], buf[5] ) );
 
 #if defined(POLARSSL_HAVE_TIME)
-    t = time( NULL );
+    /*t = time( NULL );*/
+    t = 1;
     *p++ = (unsigned char)( t >> 24 );
     *p++ = (unsigned char)( t >> 16 );
     *p++ = (unsigned char)( t >>  8 );
diff --git a/library/ssl_srv.c b/library/ssl_srv.c
index fc21754..d01332f 100644
--- a/library/ssl_srv.c
+++ b/library/ssl_srv.c
@@ -2099,7 +2099,8 @@ static int ssl_write_server_hello( ssl_context *ssl )
                    buf[4], buf[5] ) );
 
 #if defined(POLARSSL_HAVE_TIME)
-    t = time( NULL );
+    /*t = time( NULL );*/
+    t = 1;
     *p++ = (unsigned char)( t >> 24 );
     *p++ = (unsigned char)( t >> 16 );
     *p++ = (unsigned char)( t >>  8 );
diff --git a/CMakeLists.txt b/CMakeLists.txt
index bd0367d..b22a47f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -73,6 +73,7 @@ endif(ENABLE_ZLIB_SUPPORT)
 
 add_subdirectory(library)
 add_subdirectory(include)
+add_subdirectory(fuzz)
 
 if(ENABLE_TESTING)
   if(CMAKE_COMPILER_IS_GNUCC)
