--- a/library/ssl_cli.c
+++ b/library/ssl_cli.c
@@ -490,7 +490,8 @@ static int ssl_generate_random( mbedtls_ssl_context *ssl )
 #endif
 
 #if defined(MBEDTLS_HAVE_TIME)
-    t = time( NULL );
+    /*t = time( NULL );*/
+    t = 1;
     *p++ = (unsigned char)( t >> 24 );
     *p++ = (unsigned char)( t >> 16 );
     *p++ = (unsigned char)( t >>  8 );
--- a/library/ssl_srv.c
+++ b/library/ssl_srv.c
@@ -2153,7 +2153,8 @@ static int ssl_write_server_hello( mbedtls_ssl_context *ssl )
                         buf[4], buf[5] ) );
 
 #if defined(MBEDTLS_HAVE_TIME)
-    t = time( NULL );
+    /*t = time( NULL );*/
+    t = 1;
     *p++ = (unsigned char)( t >> 24 );
     *p++ = (unsigned char)( t >> 16 );
     *p++ = (unsigned char)( t >>  8 );
