--- a/library/ssl_cli.c
+++ b/library/ssl_cli.c
@@ -551,7 +551,8 @@ static int ssl_write_client_hello( ssl_context *ssl )
                    buf[4], buf[5] ) );
 
 #if defined(POLARSSL_HAVE_TIME)
-    t = time( NULL );
+    /*t = time( NULL );*/
+    t = 1;
     *p++ = (unsigned char)( t >> 24 );
     *p++ = (unsigned char)( t >> 16 );
     *p++ = (unsigned char)( t >>  8 );
--- a/library/ssl_srv.c
+++ b/library/ssl_srv.c
@@ -2099,7 +2099,8 @@ static int ssl_write_server_hello( ssl_context *ssl )
                    buf[4], buf[5] ) );
 
 #if defined(POLARSSL_HAVE_TIME)
-    t = time( NULL );
+    /*t = time( NULL );*/
+    t = 1;
     *p++ = (unsigned char)( t >> 24 );
     *p++ = (unsigned char)( t >> 16 );
     *p++ = (unsigned char)( t >>  8 );
